lib/foreman/renderer.rb

```diff
@@ -232,6 +275,11 @@ def load_template_vars
       @template_url = params['url']
     end
 
+    def global_setting(name, default = nil)
```

Comment:
what's the point of having `default` if setting already provide default? when 2 defaults are useful?

Reply:
The default of default (the option :-) ) is used in templates to have sane default if users does not opt-in and leaves it blank (that's what we ship Foreman with). What do you recommend instead?

Reply:
I still don't get it, if I use `global_settings(:some, 'value')`, I'll get value back if there's no `Setting[:some]`. But in that case, why would I call the `global_settings` in first place? I'd just use `value`. Or I'd do `global_settigs(:root_pass) || 'change_me'` instead of `global_settigns(:root_pass, 'change_me')`